wiring/src/spark_wiring_print.cpp

```diff
@@ -170,10 +170,11 @@ size_t Print::printFloat(double number, uint8_t digits)
 {
   size_t n = 0;
 
-  if (isnan(number)) return print("nan");
-  if (isinf(number)) return print("inf");
-  if (number > 4294967040.0) return print ("ovf");  // constant determined empirically
-  if (number <-4294967040.0) return print ("ovf");  // constant determined empirically
+  // FIXME: Trade off to save memory
+  //if (isnan(number)) return print("nan");
+  //if (isinf(number)) return print("inf");
+  //if (number > 4294967040.0) return print ("ovf");  // constant determined empirically
+  //if (number <-4294967040.0) return print ("ovf");  // constant determined empirically
```

Comment:
It's a good question why we use a custom implementation of the double to string conversion here, however, let's keep this code intact for now, and fix the flash overflow in the scope of another PR.